api/python/cell_census/src/cell_census/compute/highly_variable_genes.py

```diff
@@ -1,25 +1,23 @@
 import numpy as np
 import pandas as pd
-import tiledbsoma as soma
+from somacore.query.query import ExperimentAxisQuery
```

Comment:
The name is exported at the top level, so this should be:
> from tiledbsoma import ExperimentAxisQuery

Also, Google Python conventions are to never import a naked name like that, but instead import the module.  So a better style would be:

> import tiledbsoma as soma

and then use it as:

> soma.ExperimentAxisQuery

Reply:
Made a set of fixes in c89938e. A couple more fixes will be made after required tiledbsoma updates (https://github.com/single-cell-data/TileDB-SOMA/issues/791).

(I've been waffling on the correct way to import, so thank you for the style advice!)